.github/workflows/CI.yml

```diff
@@ -0,0 +1,36 @@
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+name: Continuous integration
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: install system deps
+        run: sudo apt install libdbus-1-dev libparted-dev
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+      - uses: actions-rs/cargo@v1
+        with:
+          command: check
```

Comment:
This is unnecessary as clippy runs check itself.

Reply:
this is true. i've pulled this from a larger CI pipeline from another project. the 'check' target is useful for 'failing fast'. the job will typically complete much faster than the 'clippy' target (especially on large codebases)

if 'failing' fast isn't useful, i can strip out the 'check' target